project-fortis-pipeline/ops/create-cluster.sh

```diff
@@ -118,20 +141,50 @@ echo "Finished. Now installing Spark helm chart."
   "${translationsvctoken}"
 
 echo "Finished. Verifying deployment."
-./verify-deployment.sh \
-  "${graphql_service_host}"
-
+if [ "${endpoint_protection}" == "none" ]; then
+  ./verify-deployment.sh \
+    "${graphql_service_host}"
+else
+  # request a public ip in order to verify the cluster deployment
```

Comment:
Do you think it would make sense to encapsulate this logic in the `verify-deployment.sh` script?